docs/pages/reference/backends.mdx

```diff
@@ -1467,3 +1467,92 @@ teleport:
     sync: NORMAL
     journal: WAL
 ```
+
+## CockroachDB
+
+Teleport can use [CockroachDB](https://www.cockroachlabs.com/) as a storage backend
+to achieve high available and survive regional failures. You must take steps to 
+protect access to CockroachDB in this configuration because that is where Teleport
+secrets like keys and user records will be stored.
+
+<Admonition
+  type="warning"
+  title="IMPORTANT"
+>
+  `CockroachDB` can currently only be used to store Teleport's internal database in a
+  highly-available way. This will allow you to have multiple auth servers in your
+  cluster for a high availability deployment, but it will not also store Teleport audit events
+  for you in the same way that [DynamoDB](#dynamodb) or
+  [Firestore](#firestore) will.
```

Comment:
This confused me on first read. It sounded like we were saying "be careful - you can only use this for HA and not for non-HA"
```suggestion
  CockroachDB can currently only be used to store Teleport's cluster state.
  It cannot be used for Teleport's audit log in the same way that
  [DynamoDB](#dynamodb) or Firestore](#firestore) wan.
```

I removed the backticks from CockroachDB because it is just a product name. It is not a command, code snippet, etc.